README.md

```diff
@@ -319,60 +319,60 @@ Get.context
 Get.contextOverlay
 
 // If you need a changeable height/width (like browser windows that can be scaled) you will need to use context.
-context.width
-context.height
+Get.context.width
```

Comment:
Na verdade isso está correto. O Get funciona sem context, mas esses recursos, são recursos da extensão do context usadas para janelas adaptativas, como desktop/navegador. 

context.width é similar a MediaQuery.of(context).size.width e é diferente de Get.width que dá o tamanho imutável da tela. Ao dimensionar a tela do navegador, Get.width continuará dando a mesma altura de antes, enquanto context.widget será chamado e reconstruido a cada atualização do tamanho da tela